Steamworks/src/org/usfirst/frc/team4786/robot/subsystems/DriveTrain.java

```diff
@@ -1,32 +1,60 @@
 package org.usfirst.frc.team4786.robot.subsystems;
 import org.usfirst.frc.team4786.robot.RobotMap;
+import org.usfirst.frc.team4786.robot.commands.DriveToPosition;
 import org.usfirst.frc.team4786.robot.commands.OpenLoopDrive;
 
 import com.ctre.CANTalon;
+import com.ctre.CANTalon.TalonControlMode;
+
 import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 /**
  *
  */
 public class DriveTrain extends Subsystem {
 	
-	//CANTalon objects
-	CANTalon backLeft = new CANTalon(RobotMap.backLeftPort);
-	CANTalon backRight = new CANTalon(RobotMap.backRightPort);
-	CANTalon frontLeft = new CANTalon(RobotMap.frontLeftPort);
-	CANTalon frontRight = new CANTalon(RobotMap.frontRightPort);
+	//CANTalon objects, there is a reason why they are private
+	private CANTalon backLeft = new CANTalon(RobotMap.backLeftPort);
+	private CANTalon backRight = new CANTalon(RobotMap.backRightPort);
+	private CANTalon frontLeft = new CANTalon(RobotMap.frontLeftPort);
+	private CANTalon frontRight = new CANTalon(RobotMap.frontRightPort);
+	
+	//Set Important PID Variables
+	//public int error = (int) (rawCodesPerRev * gearBoxRatio / RobotMap.ERROR_CONSTANT);
 	
 	public DriveTrain(){
+		//Enable the Talons!
+		frontLeft.enable();
+		frontRight.enable();
+		backLeft.enable();
+		backRight.enable();
+		
 		//follower code
 		backLeft.changeControlMode(CANTalon.TalonControlMode.Follower);
 		backRight.changeControlMode(CANTalon.TalonControlMode.Follower);
 		backLeft.set(RobotMap.frontLeftPort);
 		backRight.set(RobotMap.frontRightPort);
-		frontLeft.setInverted(true);
 		//this inverts the cantalons on the right side
+		frontLeft.setInverted(true);
+		
+		//Beginning of the world of PID!
+		frontLeft.setProfile(0);
+		frontRight.setProfile(0);
+		frontLeft.setPID(RobotMap.LeftP, RobotMap.LeftI, RobotMap.LeftD, RobotMap.LeftF, 0, 0, 0);		
+		frontRight.setPID(RobotMap.RightP, RobotMap.RightI, RobotMap.RightD, RobotMap.RightF, 0, 0, 0);
+		frontLeft.setCloseLoopRampRate(0.125 / 8);
+		frontRight.setCloseLoopRampRate(0.125 / 8);
```

Comment:
These hardcoded numbers need to be moved to the RobotMap class. This needs to be done before the pull request is approved.